src/app/view/components/_common/searchable-dropdown/component/searchable-dropdown.component.html

```diff
@@ -14,9 +14,8 @@
         *ngIf="label"
         (click)="searchPanel.toggle($event)"
         [label]="label"
-        icon="fa fa-caret-down"
+        icon="pi pi-chevron-down"
```

Comment:
test for then new icon an update removing the css class

Reply:
What do you mean @alfredo-dotcms ?